src/main/java/services/notification_sending/EmailNotificationPresenter.java

```diff
@@ -0,0 +1,81 @@
+package services.notification_sending;
+
+import java.util.Properties;
+import javax.mail.*;
+import javax.mail.internet.*;
+
+/**
+ * Sends out email notifications.
+ * Keeps a record of whether the user enabled the email notification settings.
+ */
+public class EmailNotificationPresenter implements NotificationPresenter, SettingsRegistry {
+    private boolean enabled;
+    private String senderEmail;
+    private String password;
+    private String userEmail;
+    private String subject;
+
+    EmailNotificationPresenter(boolean enabled, String senderEmail, String password, String userEmail, String subject) {
+        this.enabled = enabled;
+        this.senderEmail = senderEmail;
+        this.password = password;
+        this.userEmail = userEmail;
+        this.subject = subject;
+    }
+
+    @Override
+    public void presentNotification(String message) {
+        //Get properties object
+        Properties props = new Properties();
+        props.put("mail.smtp.host", "smtp.gmail.com");
+        props.put("mail.smtp.socketFactory.port", "465");
+        props.put("mail.smtp.socketFactory.class",
+                "javax.net.ssl.SSLSocketFactory");
+        props.put("mail.smtp.auth", "true");
+        props.put("mail.smtp.port", "465");
+
+        //get Session
```

Comment:
Fairly redundant comment when the next line says "Session get Instance"

Might be clearer if you describe why we need the anonymous class here (the `new javax.mail.Authenticator() {}`) and what said anonymous class is doing.